CHANGELOG.rst

```diff
@@ -1,3 +1,10 @@
+0.4.4
+  * Added support for REVERSE video and UNDERLINE.
+  * ANSI codes CSI [ 0-37 m no longer render as BRIGHT if the default terminal
+    foreground color is BRIGHT.
```

Comment:
I may have been a bit over-zealous there. Definitely agree about the last two not being in the changelog. The first two result in a change in behaviour, especially no longer getting a BRIGHT foreground colour with CSI [ 30-37 m. That should be documented somewhere, but I'm happy to remove it from the changelog.